src/cce_rpc/engine.proto

```diff
@@ -92,3 +93,20 @@ message Stats {
 message ThresholdsFile {
   string filename = 1;
 }
+
+message HostIdentifier {
+  oneof identifier {
+	string host_name = 1;
+	uint32 id = 2;
+  }
+}
+
+message EngineHost {
+  string name = 1;
```

Comment:
I think this is not coherent. host_name in HostIdentifier is the same as name in EngineHost, isn't it?
So try to keep same names for same things. Personaly, I prefer name, but if you prefer host_name, go for it!